docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -53,6 +57,16 @@ The server element enables you to easily represent a single server, a stack of s
 
 {{< figure src="/media/docs/grafana/canvas-server-element-9-4-0.png" max-width="750px" caption="Canvas server element" >}}
 
+### Button
+
+The button element enables you to add a basic button to the canvas. Button elements support triggering basic, unauthenticated API calls. API settings are found in the button element editor. You can pass template variables in the API editor.
+
+{{% admonition type="note" %}}
+To trigger an API call when clicking on a button in the canvas, [inline editing](#inline-editing) must be disabled.
```

Comment:
```suggestion
For a button click in a canvas to trigger an API call, [inline editing](#inline-editing) must be disabled.
```

Reply:
hm the wording here feels a little off - I will try to simplify this a bit